shared/params/mainnet_config.go

```diff
@@ -123,6 +123,7 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	MaxPeersToSync:            15,
 	SlotsPerArchivedPoint:     2048,
 	GenesisCountdownInterval:  time.Minute,
+	HeadOfVotingPeriodLength:  32,
```

Comment:
From the linked document:

> In mainnet configuration, the head of the voting period is the first 32 slots of the period `sqrt(1024) = 32`

Reply:
Please add a comment
```suggestion
	HeadOfVotingPeriodLength:  32, // sqrt(1024)
```